.buildkite/pipelines/pull_request/base.yml

```diff
@@ -87,7 +87,7 @@ steps:
       queue: n2-4-spot
     depends_on: build
     timeout_in_minutes: 40
-    parallelism: 2
+    parallelism: 6
```

Comment:
Can we also update 

https://github.com/elastic/kibana/blob/main/.buildkite/pipelines/es_serverless/verify_es_serverless_image.yml#L53

https://github.com/elastic/kibana/blob/main/.buildkite/pipelines/on_merge.yml#L88

Reply:
Thanks @jbudz I'll follow up on that tomorrow!!